doc/radosgw/config-ref.rst

```diff
@@ -937,6 +950,29 @@ Barbican Settings
 :Default: None
 
 
+HashiCorp Vault Settings
+========================
+
+``rgw crypt vault auth```
+
+:Description: Type of authentication method to be used. The only method
+              currently supported is ``token``.
+:Type: String
+:Default: ``token``
+
+``rgw crypt vault token file``
+
+:Description: If authentication method is ``token``, provide a path to the token
+              file, which should be readable only by Rados Gateway.
+:Type: String
+:Default: None
+
+``rgw crypt vault addr``
+
+:Description: Provide a URL to the Vault server secret path.
```

Comment:
> secret path

i think this last bit no longer applies, right? maybe call this the `base URL to the Vault server`?

Reply:
Good point I'll fix that